packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -76,6 +82,15 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
   /** The component's text. */
   @Prop({ reflect: true }) textLabel!: string;
 
+  /**
+   * The component's short label.
+   *
+   * When provided, the short label will be displayed in the component's selection.
+   *
+   * It is recommended to use 5 chars or less.
+   */
+  @Prop({ reflect: true }) textLabelShort: string;
```

Comment:
Maybe we can come up with a better name for this prop?

Something like `snippet` or `summary`? Or `selected-label`?

Reply:
I think at some point we should rename `textLabel` to just `label` as well.

Reply:
> Something like snippet or summary? Or selected-label?

When specified this prop shows an abbreviated version of the label that is used in selection. The suggested names do not quite reflect this.

> I think at some point we should rename textLabel to just label as well.

Agreed, but until then it might help associate both props by matching the existing name. There were other suggestions in the design spec, but `textLabelShort` was the shortest and most expressive (IMO). Open to alternatives. cc @SkyeSeitz @ashetland

Reply:
Sadly, I've got nothin' better than `textLabelShort`. 

Long-term I do like the idea of renaming the main two props to `label` and `description` across the system. It's not super consistent atm.

Reply:
Just throwing this out there:

Instead of adding another prop we plan on renaming in the future, why not just deprecate `textLabel` for `label` and add `description`?

Reply:
Assuming we were using `heading`/`description`. Would we go with `shortHeading`, `headingShort` or something else?

Reply:
subheading?

Reply:
> Instead of adding another prop we plan on renaming in the future, why not just deprecate textLabel for label

We'd want to use `heading` since `label` is mostly used for accessible text, right?

Also wanted to point out that my PR title isn't quite accurate. It's a short heading/label we're adding, not a description. Will update.

Reply:
@driskull [This example](https://codepen.io/jcfranco/pen/QWXjMwZ?editors=1010) shows how the prop comes into play.

Reply:
Yeah I think something like `shortHeading` should be good. 👍

Reply:
Agree with moving towards a consistent `heading` & `description` paradigm across components accompanied by `shortHeading` when relevant.

Reply:
Sounds good. Thanks for the discussion, y'all! ✨💪✨

I'll submit an issue/PR to deprecate `textLabel` separately since it is out of scope.